class A3Exception(Exception):
    pass

class TaskFailedException(A3Exception):
    pass 